src/routes/api/index.js

```diff
@@ -1,15 +1,12 @@
 import express from 'express';
 import authRouter from './auth';
+import profileRoute from './profile';
 import passwordRouter from './passwordResetRoute';
 
 const usersRouter = express.Router();
 
 usersRouter.use('/v1/auth', authRouter);
-
-usersRouter.use('/v1/users', passwordRouter);
-
-usersRouter.get('/', (req, res) => res.status(200).send('Welcome to  Shadowcat API'));
-
+usersRouter.use('/v1/profiles', profileRoute);
```

Comment:
`profiles` is a route that belongs to a `user` so this should be under the `user` route like `/v1/users/profiles`

Reply:
ok thanks a lot